src/storage/instance.ts

```diff
@@ -0,0 +1,29 @@
+import Storage from '.';
+import { ACCESS_TOKEN } from '../constants';
+
+export default class Tokens extends Storage<string> {
+  private static instance?: Tokens;
+
+  private constructor() {
+    super();
+  }
+
+  public static getInstance() {
+    if (!this.instance) {
+      this.instance = new Tokens();
+    }
+
+    return this.instance;
+  }
+
+  public getAccessToken() {
+    return this.get(ACCESS_TOKEN);
+  }
+
+  public setAccessToken(accessToken: string) {
+    this.set(ACCESS_TOKEN, accessToken);
+  }
+  public clear() {
+    this.clearItems([ACCESS_TOKEN]);
+  }
+}
```

Comment:
I'm not against of the singleton pattern, but I don't like it in Javascript. 

Every export is always a "singleton", and that's because how exports works.

Instead of making the consumer use the `getInstance` every time, you could just export the instance like so:

```suggestion
import Storage from '.';
import { ACCESS_TOKEN } from '../constants';

class Tokens extends Storage<string> {
 
  public getAccessToken() {
    return this.get(ACCESS_TOKEN);
  }

  public setAccessToken(accessToken: string) {
    this.set(ACCESS_TOKEN, accessToken);
  }
  public clear() {
    this.clearItems([ACCESS_TOKEN]);
  }
}

const tokensSingleton = new Token();
export default tokenSingleton;
```

Once more, there's nothing wrong with doing it like you guys are, just throwing some light into how exports work in JS.